app/views/katello/api/v2/subscriptions/base.json.rabl

```diff
@@ -13,3 +13,12 @@ attributes :sockets, :cores, :ram
 attributes :instance_multiplier, :stacking_id, :multi_entitlement
 attributes :type
 attributes :name => :product_name
+attributes :unmapped_guest
+attributes :virt_only
```

Comment:
@ehelms wrong line, my bad, I actually meant this one http://git.io/vW53Z Do  you think we only return virt_only and unmapped_guest if virtual and update the UI there?